app/services/cocina_physical_location.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+# Class to handle Cocina physical location information
+class CocinaPhysicalLocation
```

Comment:
👍🏻 like this choice to extract this as a class to manage the various accessors. Also glad to see `#path` getting some use here.